api/v1alpha1/httproutefilter_types.go

```diff
@@ -37,11 +37,10 @@ type HTTPRouteFilterSpec struct {
 
 // HTTPURLRewriteFilter define rewrites of HTTP URL components such as path and host
 type HTTPURLRewriteFilter struct {
-	// Hostname is the value to be used to replace the Host header value during
+	// Hostname is the value to be used to replace the Hostname header value during
```

Comment:
Nit: `HostName` Header should be `Host` header.
```suggestion
	// Hostname is the value to be used to replace the Host header value during
```